require of a missing iterator.js file breaks builds

https://unpkg.com/browse/vega-embed@6.20.0/build/vega-embed.js has a line `require('./iterator.js')(Yallist$1);` which breaks some build systems due to that file missing from the package https://unpkg.com/browse/vega-embed@6.20.0/build/.

In the previous version (https://unpkg.com/browse/vega-embed@6.19.1/build/vega-embed.js) the iterator wasn't created with a require, but as an inlined function. There's no reference to iterator in the source code, however, so I suppose this comes from some other package, or the packaging tool?

We're affected by this. We get the following error in the build pipline

```
Failed to compile.

/home/jenkins/workspace/.../node_modules/vega-embed/build/vega-embed.module.js
Module not found: Can't resolve './iterator.js' in '/home/jenkins/workspace/..../node_modules/vega-embed/build'
```

Do you know how to fix this?

This issues is also affecting us:
```WARNING in ./node_modules/vega-embed/build/vega-embed.module.js
Module not found: Error: Can't resolve './iterator.js' in '.../node_modules/vega-embed/build'
 @ ./node_modules/vega-embed/build/vega-embed.module.js
```

--- a/package.json
+++ b/package.json
@@ -51,6 +51,8 @@
     "concurrently": "^6.4.0",
     "del-cli": "^4.0.1",
     "jest-canvas-mock": "^2.3.1",
+    "patch-package": "^6.4.7",
+    "postinstall-postinstall": "^2.1.0",
     "rollup": "2.60.1",
     "rollup-plugin-bundle-size": "^1.0.3",
     "rollup-plugin-terser": "^7.0.2",
@@ -92,6 +94,7 @@
     "eslintbase": "beemo eslint .",
     "format": "yarn eslintbase --fix && yarn prettierbase --write",
     "lint": "yarn eslintbase && yarn prettierbase --check",
-    "release": "yarn build && auto shipit"
+    "release": "yarn build && auto shipit",
+    "postinstall": "patch-package"
   }
 }

--- a/patches/yallist+4.0.0.patch
+++ b/patches/yallist+4.0.0.patch
@@ -1,0 +1,15 @@
+diff --git a/node_modules/yallist/yallist.js b/node_modules/yallist/yallist.js
+index 4e83ab1..f25394e 100644
+--- a/node_modules/yallist/yallist.js
++++ b/node_modules/yallist/yallist.js
+@@ -422,5 +422,9 @@ function Node (value, prev, next, list) {
+ 
+ try {
+   // add if support for Symbol.iterator is present
+-  require('./iterator.js')(Yallist)
++  Yallist.prototype[Symbol.iterator] = function* () {
++    for (let walker = this.head; walker; walker = walker.next) {
++      yield walker.value
++    }
++  }
+ } catch (er) {}

